.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will build the project with Gradle and run integration tests
+# Because secrets are not available on external forks, this job is expected to fail
```

Comment:
So if someone submits a PR to this repo using a fork, the checks will fail? Are ther any checks that they can run in a PR?

Perhaps we can integrate mergequeue or mergify to run tests before the merge

Reply:
Merge queue would be good! Let me follow up with you offline on this.